help/en/Acknowledgements.shtml

```diff
@@ -519,6 +519,9 @@
 
         <li>Randy Jones, who provided identification info for several Quantum decoders</li>
 
+        <li>Kenneth Jost, <!-- SeaU2 -->
```

Comment:
I think this should be Kenneth Tjøstheim. See line 15 in `xml/signals/Norway2001/appearance-Distant.xml`.

Reply:
If `ø` doesn't work, use Kenneth Tjostheim.